test/fixtures/academic_calendars.yml

```diff
@@ -0,0 +1,20 @@
+one:
+  id: 1
```

Comment:
fixture'larda id belirtmemelisin! aksi halde testler bir geçer bir geçmez, ne olduğunu bile anlayamazsın :)

Reply:
https://blog.bigbinary.com/2014/09/21/tricks-and-tips-for-using-fixtures-in-rails.html